examples/nlp/token_classification/punctuation_capitalization.py

```diff
@@ -54,7 +54,7 @@
 parser.add_argument("--ignore_start_end", action='store_false')
 parser.add_argument("--ignore_extra_tokens", action='store_false')
 parser.add_argument("--none_label", default='O', type=str)
-parser.add_argument("--shuffle_data", default=True, type=bool)
+parser.add_argument("--shuffle_data", action='store_true')
```

Comment:
if default should be true, it should be store_false

Reply:
You are able to edit the files yourself if you want to update them.
But I'll make this change

Reply:
I changed them